src/covid_model_seiir_pipeline/regression/workflow.py

```diff
@@ -81,20 +79,17 @@ def get_task(self, regression_version: str):
 
 class RegressionWorkflow:
 
-    def __init__(self, regression_specification: RegressionSpecification,
-                 ode_fit_specification: FitSpecification):
-        self.regression_specification = regression_specification
-        self.ode_fit_specification = ode_fit_specification
-
+    def __init__(self, regression_version):
+        self.regression_version = regression_version
```

Comment:
This must be a YYYY-MM-DD.VV string right? Or does it include the path leading up to it?